webpack.config.js

```diff
@@ -113,7 +113,8 @@ module.exports = {
     alias: {
       css: path.join(__dirname, "/troposphere/static/css/"),
       images: path.join(__dirname, "/troposphere/static/images/"),
-      highcharts: "highcharts-commonjs"
+      highcharts: "highcharts-commonjs",
+      react: path.join(__dirname, "/node_modules/react")
```

Comment:
@jchansen I was thinking that might be the case. I just cloned the project, and ran a local install.
````npm WARN troposphere-ui@1.0.2 requires a peer of react@^15.0.1 but none was installed.```
So it looks like `npm install` doesn't install them.

But I concur that the issue is probably troposphere-ui having react in its node_modules.